### PR TITLE
Proposal: consistent file associations

### PR DESCRIPTION
**Pull request by 0-sv**
If I open values-schema.yaml I expect that it will select the _"`YAML` file association"_, and that if you open YAML files with Helm syntax that it will select the _"`helm-template` file association"_ 

BTW: the latter is not named consistently, in the "File Association menu" it shows up as `helm-template`, and in `settings.json` it shows up as `helm`).

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -32,18 +32,25 @@
   "eslint.format.enable": true,
   "eslint.validate": ["javascript", "typescript"],
   "files.associations": {
-    "*.gotmpl": "helm",
+    "*.yaml": "yaml",
     "*.tmpl": "nginx",
-    "*.tpl": "yaml",
-    "*.yml": "yaml",
```
Comment (j-zimnowoda) [inline .vscode/settings.json:38]: There are still some `*.yml` files

**Comment by Morriz** [comment]
I am not sure what is intended here. Can you be specific? It is not possible for go templating to understand the nature of what it is templating. If you think helm charts or helmfile templating should be aware of the format it is templating, then please take this upstream.

File: .vscode/settings.json [hunk 2]
```diff
@@ -32,18 +32,25 @@
...
-    ".secrets*": "shellscript",
+    "*.tpl": "helm",
+    "*.gotmpl": "helm",
+    "_*.tpl": "helm",
+    "helmfile-*.yaml": "helm",
+    "helmfile.yaml": "helm",
+    "chart/*/**.yaml": "helm",
+    "charts/*/**.yaml": "helm",
```
Comment (Morriz) [inline .vscode/settings.json:43]: please don't do this...only templates folder should match

File: .vscode/settings.json [hunk 3]
```diff
@@ -32,18 +32,25 @@
...
+    "kind/*/**.yaml": "helm",
+    "tests/*/**.yaml": "helm",
+    "values/*/**.yaml": "helm",
```
Comment (Morriz) [inline .vscode/settings.json:46]: yaml

**Review by Morriz — COMMENTED** [review]
I recommend closing this PR as it only introduces problems

**Pull request closed without merging**